im.bernard.Memorado.json

```diff
@@ -42,7 +42,7 @@
                     "type" : "git",
                     "url" : "https://github.com/wbernard/Memorado.git",
                     "tag" : "0.1",
-                    "commit" : "f5f9b1770feefaf003994328ce2b7e4483071fc7"
+                    "commit" : "52bdfcefe4352c9f49f0b496f088a1bc07e16f63"
```

Comment:
retagging IS a bad practice. This should have been a new tag.

Reply:
Ah, including for the very first release?

Reply:
Yes